docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  bbdd:
+    image: postgres
+    volumes:
+      - bbdd-data:/var/lib/postgresql
+      - ./docker/postgres:/docker-entrypoint-initdb.d
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: earthacademy
+      POSTGRES_PASSWORD: earthacademy
+      POSTGRES_DB: earthacademy
+  backend:
+    image: node:10
+    volumes:
+      - ./:/var/www/web
+    command:  bash -c "cd /var/www/web && npm install && npm start"
+    ports:
+      - "4000:4000"
+  frontend:
+    image: node:10
+    volumes:
+      - ./frontend:/var/www/web
+    command:  bash -c "cd /var/www/web && npm install && npm run dev"
```

Comment:
I think if you specify a `build` context, you don't need to do `cd /var/www/web`. Also, you don't need to use `bash -c`, what you put on the `command` will be executed on the command line.

https://docs.docker.com/compose/compose-file/#compose-file-structure-and-examples